run/django/e2e_test.py

```diff
@@ -0,0 +1,412 @@
+    yield CLOUDSQL_INSTANCE
+
+    subprocess.run(
+        [
+            "gcloud",
+            "sql",
+            "databases",
+            "delete",
+            POSTGRES_DATABASE,
+            "--instance",
+            POSTGRES_INSTANCE,
+            "--quiet",
+        ],
+        check=True,
+    )
+
+    subprocess.run(
+        [
+            "gcloud",
+            "sql",
+            "users",
+            "delete",
+            POSTGRES_USER,
+            "--instance",
+            POSTGRES_INSTANCE,
+            "--quiet",
+        ],
+        check=True,
+    )
+
+    # TODO: remove when binding is static
+    # Remove policy binding
+    """
+    subprocess.run(
+        [
+            "gcloud",
+            "projects",
+            "remove-iam-policy-binding",
+            PROJECT,
+            "--member",
+            f"serviceAccount:{project_number}@cloudbuild.gserviceaccount.com",
+            "--role",
+            "roles/cloudsql.client",
+        ],
+        check=True,
+    )
+    """
+
+
+@pytest.fixture
+def media_bucket():
+    # Create storage bucket
+    subprocess.run(
+        ["gsutil", "mb", "-l", REGION, f"gs://{CLOUD_STORAGE_BUCKET}"],
+        check=True,
+    )
+
+    yield CLOUD_STORAGE_BUCKET
+
+    # Delete storage bucket contents, delete bucket
+    subprocess.run(
+        ["gsutil", "-m", "rm", "-r", f"gs://{CLOUD_STORAGE_BUCKET}"], check=True
+    )
+    subprocess.run(["gsutil", "rb", f"gs://{CLOUD_STORAGE_BUCKET}"], check=True)
+
+
+@pytest.fixture
+def secrets(project_number):
+    # Create a number of secrets and allow Google Cloud services access to them
+
+    def create_secret(name, value):
+        secret = client.create_secret(
+            request={
+                "parent": f"projects/{PROJECT}",
+                "secret": {"replication": {"automatic": {}}},
+                "secret_id": name,
+            }
+        )
+
+        client.add_secret_version(
+            request={"parent": secret.name, "payload": {"data": value.encode("UTF-8")}}
+        )
+        print(f"DEBUG: {name}\n{value}")
+
+    def allow_access(name, member):
+        subprocess.run(
+            [
+                "gcloud",
+                "secrets",
+                "add-iam-policy-binding",
+                name,
+                "--member",
+                member,
+                "--role",
+                "roles/secretmanager.secretAccessor",
+            ],
+            check=True,
+        )
+
+    client = sm.SecretManagerServiceClient()
+    secret_key = uuid.uuid4().hex[:56]
+    settings = f"""
+DATABASE_URL=postgres://{POSTGRES_USER}:{POSTGRES_PASSWORD}@//cloudsql/{PROJECT}:{REGION}:{POSTGRES_INSTANCE}/{POSTGRES_DATABASE}
+GS_BUCKET_NAME={CLOUD_STORAGE_BUCKET}
+SECRET_KEY={secret_key}
+    """
+
+    create_secret(SECRET_SETTINGS_NAME, settings)
+    allow_access(
+        SECRET_SETTINGS_NAME,
+        f"serviceAccount:{project_number}-compute@developer.gserviceaccount.com",
+    )
+    allow_access(
```

Comment:
It should be fine to set these statically by setting secret accessor roles for on these SAs